src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -272,6 +275,14 @@ export class CellModel extends Disposable implements ICellModel {
 		return this._options.notebook.language;
 	}
 
+	public get savedConnectionName(): string | undefined {
+		return this._savedConnectionName;
+	}
+
+	public get context(): ConnectionProfile | undefined {
```

Comment:
What's this being added for? It doesn't seem to be used anywhere here.

Reply:
This is used for backend multi-connection work but I wanted this PR to just be focused on the metadata. Thanks for catching that, I'll remove it for now.